### ISSUE TITLE
Spelling Errors on Discord RPC Godot 'Getting Started' Page

### ISSUE THREAD

**Issue by diamonddevv** [vaporvee/docs#1]
There are a couple of spelling mistakes on the 'Getting Started' page.

Nothing hugely major, easily done :)
_Current Page:_
![image](https://github.com/user-attachments/assets/f966c853-9704-44be-91e1-e96490fb241e)

The correct text would be (including markdown formatting):
```
This happens normally inside the Autoload which should be automatically added by the plugin.
Something that does **not get added automatically** is your `app_id`. It is also mandatory to be able to use anything in the plugin.
```
 
In addition, the text above the code box above this text should probably have a comma in and a colon after it:
![image](https://github.com/user-attachments/assets/bbf74aa6-b3d3-4ddd-972b-a85200587d6a)
```
To make anything work in the plugin, make sure to run:
```

The `.mdx` file is located at `src/content/docs/discord-rpc-godot/index.mdx`

:)

**Comment by vaporvee** [comment]
not my first language lol

**Comment by vaporvee** [comment]
fixed in fe94e1d5a1b4f5b9541bd71b75c4f45ff1ba70d2
